x-pack/plugins/canvas/public/components/render_to_dom/index.ts

```diff
@@ -4,16 +4,4 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export class ElementHandlers {
-  resize() {}
-
-  destroy() {}
-
-  onResize(fn) {
-    this.resize = fn;
-  }
-
-  onDestroy(fn) {
-    this.destroy = fn;
-  }
-}
+export { RenderToDom } from './render_to_dom';
```

Comment:
what a weird diff from github here lol

Reply:
I know... no clue where that's coming from.  Gonna ask ops if there's a reason, or something to fix.